src/libcore/hash/mod.rs

```diff
@@ -271,21 +273,21 @@ impl<H: Default + Hasher> BuildHasher for BuildHasherDefault<H> {
     }
 }
 
-#[stable(since = "1.7.0", feature = "build_hasher")]
-impl<H> Clone for BuildHasherDefault<H> {
-    fn clone(&self) -> BuildHasherDefault<H> {
-        BuildHasherDefault(marker::PhantomData)
+#[stable(since = "1.9.0", feature = "core_impl_debug")]
+impl<H> fmt::Debug for BuildHasherDefault<H> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.pad("BuildHasherDefault")
     }
 }
 
 #[stable(since = "1.7.0", feature = "build_hasher")]
-impl<H> Default for BuildHasherDefault<H> {
-    fn default() -> BuildHasherDefault<H> {
+impl<H> Clone for BuildHasherDefault<H> {
+    fn clone(&self) -> BuildHasherDefault<H> {
         BuildHasherDefault(marker::PhantomData)
     }
 }
 
-//////////////////////////////////////////////////////////////////////////////
+/// ///////////////////////////////////////////////////////////////////////////
```

Comment:
I rearranged the functions a bit to make more sense to me, but I've updated and changed it back.